src/templates/angular/src/app/pages/users/users.component.ts

```diff
@@ -31,19 +37,34 @@ export class UsersComponent implements OnInit {
   <% if(isCrudWithNode){%>
   getUsers() {
     this.apiService.getEmployees().subscribe((res: any) => 
-      this.users = res
+      this.users = res.map((each: any)=> {
+        const refactoredUserData: object = {
+          id: each.id,
+          name: each.name,
+          username: each.username,
+          email: each.email,
+        }
+        return refactoredUserData
+        }
+      )
     )
   }
   <%}%>
   
+  <% if(isCrud || isCrudWithNode){%>
   onClickAddUser() {
     $('#addUser_modal').modal('show');
     this.data = {};
   }
 
+  public handleUserActions = (name: any, user: any) => {
+    if(name == EDIT) this.editUser(user)
+    else this.deleteUser(user.id)
```

Comment:
on else delete?
is there only two possible operation, lets specify name as well for delete, we shouln't delete in some else block, without specify operation action